css/styles.css

```diff
@@ -0,0 +1,378 @@
+/****css reset****/
```

Comment:
Nice usage of the normalizing CSS. You have an opportunity to break these files up for easier management, like putting all of your resets into a `reset.css` file that is imported via `<link>` first (after your google fonts!), followed by a `grid.css` that has all those 12-column grid system styles linked second, and then `styles.css` for all your custom styles and any overrides you want to make.